chain/jsonrpc/src/lib.rs

```diff
@@ -384,6 +384,9 @@ impl JsonRpcHandler {
                 process_method_call(request, |params| self.send_tx_commit(params)).await
             }
             "chunk" => process_method_call(request, |params| self.chunk(params)).await,
+            "congestion_info" => {
```

Comment:
We must follow the pattern and have the prefix `EXPERIMENTAL_` for all new methods we add. Please add the prefix and place this line(s) alphabetically near other experimental methods in this match

Reply:
I see, thanks for pointing it out. Should be fixed now.

Reply:
Thank you! It was faster than Lightning McQueen ❤️

Reply:
about as fast as your review :wink: